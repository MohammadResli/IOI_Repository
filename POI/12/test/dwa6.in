50
44 3 16 24 30 28 49 41 7 39 27 8 36 34 20 19 44 47 40 45 29 35 22 46 32 17 42 12 15 23 13 21 18 26 31 11 2 5 37 25 10 4 43 50 33
45 42 35 26 36 15 20 13 17 32 3 41 50 7 11 1 37 5 45 46 18 8 44 29 34 28 47 31 9 21 16 27 4 24 23 49 19 33 22 10 40 38 14 39 30 48
43 9 50 10 33 27 34 14 6 22 42 43 30 32 1 45 18 29 19 46 23 17 15 21 25 24 36 20 37 12 28 49 40 44 2 38 48 41 11 4 8 35 26 47
45 6 9 1 24 8 28 25 26 19 3 30 41 49 12 38 22 43 15 16 11 46 17 14 39 37 48 42 29 33 45 34 7 18 21 31 20 32 5 44 10 40 2 23 35 47
42 23 42 33 44 39 11 22 46 12 26 16 28 25 1 15 40 14 29 17 6 13 8 21 4 45 43 34 27 47 18 10 19 35 2 20 7 32 48 24 38 30 49
45 50 37 16 8 44 38 17 46 19 33 25 3 47 41 13 49 7 20 5 42 23 26 11 48 10 31 21 24 27 30 15 22 35 18 40 29 43 34 12 36 32 9 39 4 28
41 21 42 43 49 38 31 19 39 10 47 40 48 9 27 32 20 30 11 16 25 50 26 1 2 35 6 14 5 37 8 28 4 34 33 46 44 45 36 41 22 13
41 39 3 18 36 4 6 1 20 49 9 16 40 19 46 12 25 11 5 41 21 28 30 7 24 17 23 29 2 27 32 15 33 26 43 34 22 10 44 13 37 38
45 35 13 38 10 34 47 15 31 45 26 14 30 48 39 36 41 25 2 18 22 43 40 37 24 8 21 17 32 29 44 12 23 4 49 20 33 3 6 11 19 7 27 42 50 46
42 4 47 24 49 15 27 44 23 34 38 18 14 19 8 21 17 16 12 50 7 36 33 28 32 22 1 26 42 45 25 43 46 6 30 13 2 39 48 40 5 3 9
44 50 4 37 26 27 45 18 42 7 6 2 40 25 39 16 22 14 49 3 31 29 35 48 32 47 30 23 28 44 5 1 17 15 9 8 24 46 33 34 13 41 12 38 43
41 29 31 15 16 21 50 47 30 24 34 19 9 41 35 46 25 42 8 48 27 14 10 32 40 23 11 38 28 4 36 3 13 1 45 20 6 49 5 26 33 44
43 49 12 32 8 9 50 14 47 36 16 15 45 6 2 28 43 18 1 37 41 48 42 22 7 23 33 46 35 30 20 38 29 21 26 24 34 27 5 11 40 44 10 17
46 10 27 32 35 37 44 39 45 22 13 30 33 15 11 25 21 5 29 12 19 50 48 31 36 9 40 41 16 49 46 17 28 18 3 7 20 38 23 43 42 47 24 4 34 2 26
41 40 11 16 41 29 31 6 3 1 33 4 43 27 20 14 9 26 48 2 37 10 50 32 44 47 5 25 17 38 12 18 39 13 23 21 30 46 8 24 45 49
44 35 33 40 39 38 36 20 44 46 1 43 18 48 41 26 28 30 4 2 42 29 7 21 50 19 49 27 32 37 47 15 12 34 10 13 31 8 6 24 5 14 11 23 22
42 4 18 48 29 33 22 47 21 23 14 25 28 6 37 46 3 24 8 42 5 44 49 9 30 50 38 15 13 31 1 41 35 11 20 39 36 2 45 27 10 19 34
43 49 20 45 43 34 29 22 47 14 19 41 36 24 50 30 40 5 4 13 10 15 1 38 42 6 21 46 11 39 17 3 9 8 25 35 28 48 27 32 23 33 2 16
43 40 26 36 4 49 46 10 12 44 32 1 16 43 30 9 29 22 7 38 21 8 20 14 27 2 48 18 6 5 41 34 45 28 17 47 25 42 33 24 37 35 23 3
45 14 40 12 28 35 29 17 19 45 25 27 41 24 32 48 15 44 4 22 39 9 23 30 8 5 37 2 13 7 46 43 38 34 16 42 33 6 36 1 47 50 26 18 21 3
48 48 31 42 38 12 49 46 22 39 43 10 28 25 32 41 14 9 36 45 20 34 35 7 19 47 24 50 5 30 16 3 17 18 27 15 1 23 13 44 29 37 40 8 33 26 6 4 2
43 43 26 1 19 44 3 28 7 16 8 13 5 31 45 37 35 36 11 23 46 38 6 2 25 17 27 47 10 42 14 49 34 39 40 20 21 4 33 48 18 50 9 41
46 31 30 17 12 8 9 40 2 29 22 14 47 39 50 4 21 20 35 5 24 10 33 41 36 3 16 42 11 38 13 19 48 15 1 26 49 46 18 6 32 25 34 27 45 28 44
41 18 20 4 9 12 35 15 45 11 25 6 38 1 10 48 14 29 47 17 16 28 21 19 23 39 37 3 50 30 13 5 2 40 49 46 8 36 26 32 34 43
43 12 32 9 20 27 40 5 17 33 48 46 41 26 45 37 35 4 39 15 31 29 36 24 28 47 21 3 42 50 10 1 19 18 11 14 34 6 23 43 30 22 8 7
46 27 29 35 45 33 9 6 38 5 16 28 19 48 12 7 40 30 11 13 43 34 14 23 1 4 8 24 20 32 10 36 44 21 25 31 42 15 39 2 41 3 49 50 22 37 46
42 38 26 21 32 8 1 16 49 42 17 10 30 2 29 13 25 14 47 45 11 40 43 50 7 9 22 48 6 19 3 36 23 20 44 33 5 15 28 12 18 35 31
43 12 46 2 24 3 6 27 31 37 8 10 39 50 30 43 47 5 26 13 45 18 1 7 19 11 36 16 17 21 22 25 41 4 35 23 33 14 20 48 40 38 49 29
44 1 18 31 19 32 8 49 28 2 48 41 42 33 45 25 6 12 46 11 16 14 3 27 21 17 15 13 47 5 26 39 38 44 50 23 24 4 20 9 37 40 34 43 35
44 18 33 47 11 26 45 13 2 10 41 1 7 36 14 25 6 39 19 23 20 40 50 8 28 38 9 43 27 46 12 4 5 49 37 17 15 31 35 16 48 32 3 21 24
39 48 2 29 37 49 23 25 15 34 33 27 6 12 1 22 43 42 7 44 16 21 30 46 41 32 11 40 47 28 14 36 50 9 45 39 4 38 17 26
46 39 26 21 27 2 13 10 45 50 40 19 5 49 31 41 44 14 47 25 7 38 46 34 11 9 30 1 18 12 23 3 33 29 8 6 37 42 43 35 24 15 36 48 20 4 16
46 2 21 20 25 18 4 16 50 26 39 14 30 7 42 1 6 35 38 48 46 29 41 3 43 10 13 47 8 28 37 40 27 49 23 32 5 19 31 36 12 9 17 34 11 15 22
42 46 23 1 39 35 50 18 5 13 21 7 38 14 12 2 10 16 19 49 47 43 36 48 6 9 42 8 33 17 40 20 25 31 44 22 26 11 29 3 24 32 4
44 48 50 26 37 38 4 6 33 2 34 18 36 5 1 43 11 30 47 12 39 24 7 13 40 16 20 27 28 9 42 19 17 23 41 49 25 22 21 45 46 32 29 3 14
44 3 40 13 43 26 49 10 25 38 31 24 1 14 20 17 41 37 32 9 21 50 30 16 44 35 22 48 39 6 7 2 18 23 27 45 8 46 47 34 28 42 19 12 33
41 2 17 38 36 45 22 3 33 9 44 35 43 48 13 7 4 1 31 47 15 8 28 40 24 19 30 14 6 26 32 49 29 50 20 11 42 21 25 39 16 41
47 5 48 16 44 13 50 4 35 42 24 32 8 14 46 6 26 34 41 45 39 20 27 2 47 23 3 17 33 36 10 7 15 43 12 49 29 37 18 40 21 31 11 30 9 22 28 19
41 25 44 45 36 9 4 48 29 38 5 15 35 2 16 33 34 17 11 40 37 26 46 10 22 18 1 28 31 47 7 6 24 49 41 8 30 23 32 14 21 20
47 35 2 19 34 44 16 29 21 12 3 46 5 1 18 6 36 32 26 49 30 25 33 50 22 20 23 4 39 8 11 45 7 24 38 13 15 9 41 28 42 10 48 27 43 31 14 37
44 29 49 14 20 6 45 32 9 26 3 16 13 7 47 4 18 1 44 37 31 40 23 36 25 35 21 8 33 28 46 22 42 2 43 48 11 12 39 17 30 19 50 38 15
42 10 17 26 16 40 12 21 4 14 43 2 11 22 45 44 27 33 49 35 3 50 31 7 9 32 19 38 20 46 34 1 29 5 18 47 23 6 25 41 36 13 37
42 24 14 7 31 27 9 50 13 45 44 35 34 20 6 21 41 25 26 1 28 37 36 22 8 42 40 3 5 47 18 4 30 11 32 15 10 16 33 38 29 19 49
42 9 45 43 46 3 17 27 48 5 36 42 15 40 37 10 29 13 41 19 39 12 49 1 14 34 16 32 11 23 22 8 6 38 26 20 47 21 50 2 7 4 31
42 10 2 41 11 24 35 13 15 17 36 43 39 40 3 38 22 48 42 44 23 7 29 26 30 50 21 27 25 1 12 5 31 19 18 4 20 28 9 14 37 32 49
45 29 34 16 30 9 8 47 19 2 40 32 35 48 13 22 3 5 25 24 6 38 10 36 18 17 31 42 28 4 49 11 44 7 14 26 12 41 20 15 50 23 21 39 1 33
45 32 12 14 22 15 4 37 20 6 9 34 49 7 28 2 13 18 31 30 11 24 5 38 19 21 41 44 1 36 39 35 29 50 3 10 42 48 46 25 23 17 33 16 43 27
45 40 49 19 37 44 4 20 16 33 5 25 3 45 18 12 39 29 23 28 36 21 17 50 41 13 35 31 38 27 26 46 32 30 47 2 14 22 15 10 7 24 9 6 34 11
47 33 36 40 5 11 42 44 21 37 47 13 45 18 35 38 15 9 28 16 12 26 46 22 30 7 6 4 10 2 34 39 32 27 31 19 17 41 24 48 23 3 1 29 14 8 43 50
44 47 21 46 18 25 28 20 40 12 9 35 1 26 32 6 36 14 2 43 3 48 34 49 37 33 27 29 42 22 45 16 13 24 15 30 17 41 38 31 10 7 23 44 11
