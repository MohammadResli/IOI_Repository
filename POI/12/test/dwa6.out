30
3 4 10 11 12 13 15 16 19 21 22 23 25 26 27 29 30 32 35 38 39 40 43 44 45 46 47 48 49 50 
